faq.md

```diff
@@ -42,9 +36,9 @@ To grow demand for newly launched stablecoins, governance can offer staking cont
 
 ## What collateral are accepted by Angle?
 
-The idea is to start by being conservative and only accepting a few, but carefully chosen, collaterals. At launch, agEUR will be backed by USDC and DAI. Next, wETH will be accepted as collateral as well.
+Currently, USDC, DAI, FRAX, and FEI can be used to mint agEUR.
 
-Governance is the entity able to whitelist new collateral types.
+We are working on making wETH and wBTC available as collateral.
```

Comment:
Not true here for the current module.

Should keep the sentence about governance